examples/matRad_example8_fineSampling.m

```diff
@@ -0,0 +1,117 @@
+%% Example: Standrad Proton Treatment Plan recalculated with fine sampling 
```

Comment:
The example looks good. But I am reconsidering including it in one of the proton examples that already exist. Any opinions on that?

Reply:
Yes, I think it makes sense to include it into the already existing example 5. However, it could also be slightly confusing for people trying to understand the basic principle of simple matRad proton dose calculation. Nevertheless, I'll look for a way to include it reasonably into the example 5.

Reply:
I will include it in the already existing example 6, for recalculating the resultGUI_noise struct.